go/private/rules/library.bzl

```diff
@@ -31,10 +31,6 @@ load(
     "GoLibrary",
     "INFERRED_PATH",
 )
-load(
-    "//go/private/rules:transition.bzl",
-    "non_go_transition",
```

Comment:
Could you rebase onto master? I fixed a test that should not (rightly) fail without `non_go_transition` on the attrs of go_library (while removing it from go_binary and go_test is fine with edge transitions).

Reply:
Aha, yep, this test now fails - re-added the `non_go_transition` to `go_library` :)